src/Invio.Immutable/NullablePropertyHandler.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using System.Diagnostics;
+using System.Reflection;
+
+namespace Invio.Immutable {
+    /// <summary>
+    ///   Wraps a <see cref="IPropertyHandler" /> that handles value type properties
+    ///   to support properties of type <see cref="Nullable{TProperty}" />
+    /// </summary>
+    /// <typeparam name="TProperty">
+    ///   The type that the inner <see cref="PropertyHandlerBase{TProperty}" /> supports.
```

Comment:
*inner `IPropertyHandler` supports.

It may not inherit from `PropertHandlerBase{TProperty}`